### DISCUSSION TITLE
Custom DNS server query for specific domain ?

### DISCUSSION THREAD

**Discussion by blackyzero** [AdguardTeam/AdGuardHome#5046] [Q&A] [answered]
Hello,

How could i set custom dns server for specific domain or dns query condition? 
ex: 
dns query for "abc.com" will use dns server 8.8.8.8
xyz.com will use dns server 1.1.1.1

prefix1.abc.com will use dns server 192.168.0.1
prefix2.abc.com will use dns server 10.0.0.1

Thank you.

**Comment by ainar-g** [comment, marked as answer]
This is documented in the Wiki.

<https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#upstreams-for-domains>

**Comment by blackyzero** [comment]
yeah, that's exactly what i need.
thank you @ainar-g
